README.md

```diff
@@ -1,8 +1,8 @@
 PHP ClickHouse wrapper
```

Comment:
I still am persuaded this is more like client and not a wrapper as it doesn't wrap anything ;)